migrations/default/1_create_tables/up.sql

```diff
@@ -0,0 +1,333 @@
+drop schema if exists db cascade;
+create schema db;
+
+-------------------------------------------
+-------- lookup tables --------------------
+-------------------------------------------
+create table db.road_types (
+    id integer primary key,
+    description text not null,
+    owner text
+);
+
+create table db.unit_types (
+    id integer primary key,
+    description text not null,
+    owner text not null
+);
+
+insert into db.road_types (id, description, owner) values
+(1, 'alley', 'cris'),
+(2, 'collector', 'cris'),
+(3, 'arterial', 'cris'),
+(4, 'highway', 'cris'),
+(5, 'other', 'cris'),
+(90000, 'urban trail', 'vz');
```

Comment:
Just very briefly, I think we should try to never specify the values of our `id` columns. I've been bit in the past where I've done this and failed to reset the sequence in the postgres database. It's possible to do that, but the fact you have to do, I think, enforces the idea that the value of the `id` should be the databases job to come up with.